README.rst

```diff
@@ -69,25 +69,25 @@ Requirements and Tested Platforms
     - 3.8-3.14. Note that Python 3.10.0 contains a bug making it unsupportable by
       PyInstaller. PyInstaller will also not work with beta releases of Python
       3.15.
-- Windows (32bit/64bit/ARM64):
+- Windows (32-bit/64-bit/ARM64):
     - PyInstaller should work on Windows 7 or newer, but we only officially support Windows 8+.
-    - Support for Python installed from the Windows store without using virtual
+    - Support for Python installed from the Windows Store without using virtual
       environments requires PyInstaller 4.4 or later.
 - Linux:
     - GNU libc based distributions on architectures ``x86_64``, ``aarch64``,
       ``i686``, ``ppc64le``, ``s390x``.
     - musl libc based distributions on architectures ``x86_64``, ``aarch64``.
     - ldd: Console application to print the shared libraries required
       by each program or shared library. This typically can be found in
-      the distribution-package `glibc` or `libc-bin`.
+      the distribution package `glibc` or `libc-bin`.
     - objdump: Console application to display information from
       object files. This typically can be found in the
-      distribution-package `binutils`.
+      distribution package `binutils`.
```

Comment:
Perhaps this should have been "distribution-packaged"?

Reply:
`binutils` is often the package name.

In German, "distribution package" would be "Distributions-Paket". So, if that was written by a german, that's why the dash crept in there.

Reply:
85 and 87 have the same phrase (one without dash, one with dash), the fix in this PR is just making them identical by removing the dash.

Reply:
Similar phrase also in 82.

Reply:
So I believe that just removing the "-" is the minimal and correct fix.

Reply:
Ah, yeah, sorry. I somehow managed to overlook the other two occurrences, and the fact that you fixed all three of them.